### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by domitry**
Add installation methods for Windows and Mac.
Related to #50 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,13 +19,44 @@ At first install IPython/Jupyter. I recommend an installation using virtualenv.
...
+
+    gem install ffi-rzmq
+    gem install iruby
+
+Now you can run iruby with:
+
+    iruby notebook
+
+#### Mac
+I reccomend an installation using [Anaconda](https://store.continuum.io/cshop/anaconda/).
```
Comment (minad) [inline README.md]: typo

File: README.md [hunk 2]
```diff
@@ -7,7 +7,9 @@ This is a Ruby kernel for IPython/Jupyter and is part of [SciRuby](http://scirub
 ![Screenshot](https://cloud.githubusercontent.com/assets/50754/7956845/3fa46df8-09e3-11e5-8641-f5b8669061b5.png)
 
 ### Quick start
+The installation instructions are dividid according to environments mainly because of ZeroMQ.
```
Comment (minad) [inline README.md]: divided

File: README.md [hunk 3]
```diff
@@ -17,13 +19,44 @@ At first install IPython/Jupyter. I recommend an installation using virtualenv.
...
+    gem install ffi-rzmq
+    gem install iruby
+
+Now you can run iruby with:
+
+    iruby notebook
+
+#### Mac
+I reccomend an installation using [Anaconda](https://store.continuum.io/cshop/anaconda/).
+I has not chacked the installation to MacOS X, but four lines below were necessary in v0.1.x.
```
Comment (minad) [inline README.md]: typo, please be more careful or use a spellchecker :)

**Comment by minad** [comment]
Thx for the instructions! great! Do you have access to all those platforms?

**Comment by domitry** [comment]
Sorry for many typo ... Yes I can try IRuby on Windows, Ubuntu and Mac but I do not have the latest MacOS X so I checked on the others.

**Pull request merged**